### PR TITLE
not true

### PR DESCRIPTION
**Pull request by kruzda**

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-b557b37756/how-to/)

**Comment by DonNateR** [comment]
@kruzda: Thanks for letting us know. I'm going to confirm this on our end, and then I'll merge.

File: content/cloud-servers/rackspace-cloud-essentials-cloud-server-image-limitations.md [hunk 1]
```diff
@@ -101,9 +101,6 @@ there are a number of practical limitations in place:
 
 ### Windows-specific limitations
 
--   There are no limits on image disk size for Standard Windows Cloud
-    Servers at this time. General Purpose and I/O servers cannot be built from images with system volumes larger than 40GB.
```
Comment (hdansou) [inline content/cloud-servers/rackspace-cloud-essentials-cloud-server-image-limitations.md]: We should add the following: "Cloud servers cannot be built from images with system volumes larger than the disk size of the Cloud Server flavor selected to deploy the new server. For example,  I/O servers cannot be built from images with system volumes larger than 40GB."

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-0f7e44ea1d/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-8dd7292005/how-to/). :bow:

**Comment by kmsholcomb** [comment]
Hi @DonNateR. Edits are attached. Please let me know if you have any questions. Thanks!

[Cloud Servers image limitations_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/642811/Cloud.Servers.image.limitations_editsKH.pdf)
[Cloud Servers image limitations_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/642812/Cloud.Servers.image.limitations_editsKH.docx)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-8c1c3e4963/how-to/).

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-913e6127f3/how-to/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-6011960b0a/how-to/). :bow:

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-57a80a1161/how-to/)

**Comment by stephamon** [comment]
👍

**Pull request merged**